charts/logging-operator/templates/userrole.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if .Values.rbac.enabled }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "logging-operator.fullname" . }}-user
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
```

Comment:
```suggestion
    rbac.authorization.k8s.io/aggregate-to-admin: "true"
    rbac.authorization.k8s.io/aggregate-to-edit: "true"
```

Reply:
Looks like the edit ClusterRole is already aggregated to admin, I guess adding `aggregate-to-edit` here should be sufficient